File: mycollab-mobile/src/main/java/com/esofthead/mycollab/mobile/module/user/event/UserEvent.java
package com.esofthead.mycollab.mobile.module.user.event;

import com.esofthead.mycollab.eventmanager.ApplicationEvent;

/**
 * @author MyCollab Ltd.
 * @since 3.0
 */
public class UserEvent {
	public static class PlainLogin extends ApplicationEvent {
		private static final long serialVersionUID = 1L;

		public PlainLogin(Object source, Object data) {
			super(source, data);
		}
	}
}

File: mycollab-web/src/main/java/com/mycollab/module/user/event/UserEvent.kt
package com.mycollab.module.user.event

import com.mycollab.events.ApplicationEvent

/**
 * @author MyCollab Ltd
 * @since 6.0.0
 */
object UserEvent {
    class PlainLogin(val username: String, val password: String, val isRememberMe: Boolean)

    class Search(source: Any, data: Any?) : ApplicationEvent(source, data)

    class GotoList(source: Any, data: Any?) : ApplicationEvent(source, data)

    class GotoAdd(source: Any, data: Any?) : ApplicationEvent(source, data)

    class GotoRead(source: Any, data: Any?) : ApplicationEvent(source, data)

    class GotoEdit(source: Any, data: Any?) : ApplicationEvent(source, data)
}